.travis.yml

```
@@ -1,3 +1,16 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.7.1
+  - 2.6
+  - 2.5
+  - 2.4
```

```suggestion
  - 2.4
  - jruby
  - truffleruby
```

If the suite passes, why not?